docs/src/components/HeadingCopyButton.astro

```diff
@@ -22,29 +22,38 @@
       }
 
       h.appendChild(copyButton)
+      h.classList.add('with-copy')
     })
-    window.ADD_COPY_BUTTONS()
+  window.ADD_COPY_BUTTONS()
 </script>
 <style is:global>
   button.copy:before {
     content: unset;
   }
   button.copy {
-    display: inline;
+    display: none;
     border: none;
     background: unset;
     margin: 0;
     padding: 0;
     min-width: unset;
     margin-left: 1rem;
-    font-size: 1rem;
-    line-height: 1rem;
+    font-size: 1.2rem;
+    font-weight: 500;
     height: 1rem;
-    position: relative;
-    top: -0.5rem;
+    cursor: pointer;
   }
+
   h1 button.copy,
   h2 button.copy {
-    top: -1rem;
+    font-size: 1.3rem;
```

Comment:
Check `h1` button size, it is too small

```suggestion
    font-size: 1em;
```

Reply:
Done! This is how different headings looks like: [h1](https://github.com/artalar/reatom/assets/25587090/0227be4c-dd9c-44de-91d1-ad3ae49b7ed2), [h2](https://github.com/artalar/reatom/assets/25587090/e97519c1-ea1f-435c-aea5-b8d1c86dbb78), [h3](https://github.com/artalar/reatom/assets/25587090/d05167bd-8a12-40e8-bb5c-8cc17770db32)